### DISCUSSION TITLE
Projektregeln

### DISCUSSION THREAD

**Discussion by Qiriba** [Qiriba/Rezepte-for-Dummies#3] [General]
- Sprintlänge ist 2 Wochen
- Sprint-Change-Meeting Dauer beläuft sich auf 15 min
- Alle 3 Tage
- Daily Stand Up dauert 10 min
- Team besteht aus:
      - Luca Knaus
      - Tobias Maaßen: Product Owner
      - Marcel Rohde: Scrum Master
 - Die Android App wird in Android Studio entwickelt

- Die App soll es jedem ermöglichen ganz simpel und einfach Rezepte zu kochen, sodass jeder diese nachkochen kann egal wie viel Erfahrung man selber mitbringt. Außerdem soll es möglich sein dem Benutzer die Rezeptschritte anzuzeigen und mit einer Sprachausgabe abspielen zu lassen.

**Comment by Bjoern04** [comment]
Hinzufügen:
-Vision 
-Aufgabenbereiche der einzelnen Mitglieder
-Sprint 1 Backlog (Screenshot)
-genutzte Techniken und Plattformen 

Scrum-Setup gut dargestellt

**Comment by HoshizawaYuriko** [comment]
Zu den Projektregeln kann man noch folgende Informationen hinzufügen damit alles klarer definiert ist:

- Startdatum des ersten Sprints -> Dann weiß man zu welchem Wochentag der erste Sprint gestartet hat und der Startpunkt bzw. das Ende aller nachfolgenden Sprints ist ersichtlich
- Spezifische Wochentage an dem die Dailys stattfinden z.B. jeden Montag, Donnerstag, Samstag (außer Dailys sollen wirklich immer alle 3 Tage stattfinden -> Daily könnte dann an dem selben Tag eines Sprint-Change-Meeting liegen)
- Spezifische Uhrzeit für Daily Standup festlegen falls nötig

**Comment by hmi95** [comment]
Ein Satz ist für eine Vision etwas "dünn", da kommt die Begeisterung nicht so gut rüber. Das Sprint 1 Backlog fehlt. 
Zu den verwendeten Technologien finde ich nur die Angabe "Die Android App wird in Android Studio entwickelt". Wie kommen die Rezepte in die App? Oder anders gefragt kommt die App ganz ohne einen Server aus, der sie mit Daten (Rezepten, ....) versorgt?
